### DISCUSSION TITLE
Invalid copilot token: missing token: 403

### DISCUSSION THREAD

**Discussion by nima265** [community/community#60114] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I have a GitHub Pro account that access me to the GitHub Copilot but today when I wanted to sign in to it with Visual Studio I got this error 
How can I fix it?
[INFO] [auth] [2023-07-05T08:22:32.376Z] Invalid copilot token: missing token: 403 
Auth Status: NotAuthorized
You don't have access to GitHub Copilot.

**Comment by davecheney** [comment]
@nima265 i can see in your telemetry that you are have signed in and are receiving completions.

**Reply by nima265** [reply, marked as answer]
yeah Thanks, after a while of trying, its finally ok

**Comment by Arden27** [comment]
Having the same issue. Reinstalling extension, re-login to git, reloading with "Developer: Reload Window" in vsCode - nothing helped.

2024-04-16 22:45:02.223 [info] [auth] Invalid copilot token: missing token: 403 
2024-04-16 22:45:02.692 [error] [auth] Extension activation failed: "Contact Support. You are currently logged in as Arden27."

Please help.

**Reply by davecheney** [reply]
I am sorry, we cannot assist you with this issue via this forum. Please contact support https://support.github.com/contact
